runhouse/resources/hardware/kubernetes_cluster.py

```diff
@@ -0,0 +1,108 @@
+import copy
+import logging
+import os
+import subprocess
+import warnings
```

Comment:
looks like there are some unused imports here

if you run the linter as well, will make clear which can be removed

`pre-commit run --all-files`